src/back/msl/writer.rs

```diff
@@ -1778,7 +1774,6 @@ impl<W: Write> Writer<W> {
             for (handle, var) in module.global_variables.iter() {
                 if needs_array_length(var.ty, &module.types) {
                     let idx = handle.index();
-                    self.runtime_sized_buffers.insert(handle, idx);
```

Comment:
oh right. I think this was intended to be different originally, but now it's redundant indeed